src/utils/internalState.ts

```diff
@@ -17,7 +17,8 @@ interface FormInternalState {
   products: Record<string, SimplifiedProduct>;
   cart: Cart;
   collaborator: Collaborator;
-  formName: string;
+  formName: string; // TODO: remove support for formName
```

Comment:
where is formName still needed for internal state?

Reply:
i don't think it is needed since we set `formName` for the dev nav bar in local state - removed it.